pkg/controller/kube/namesapce.go

```diff
@@ -105,7 +107,7 @@ func (c *namespaceController) createBootstrapConfigMap(ns string) {
 		"custom_bootstrap.json": GetBootstrapConfig(c.AerakiAddr, c.AerakiPort),
 	}
 	if err := c.Client.Create(context.TODO(), cm, &controllerclient.CreateOptions{
-		FieldManager: constants.AerakiFieldManager,
+		FieldManager: constants.AerakiFieldManager + "-" + c.rootNamespace,
```

Comment:
There're two problems here:

* The change of FieldManager will break existing Aeraki deployment since old resources were created with “constants.AerakiFieldManager"
* I prefer to get FiledManager value from a central place rather than passing rootNamespace around everywhere